RocketApp/app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I think the TA mentioned maybe we should share this JSON file privately, maybe we can add this in the .gitignore instead? What do you think?

Reply:
Yeah I think that is probably a good idea, I will add it to the gitignore.